### ISSUE TITLE
Numbers of positives Nyi

### ISSUE THREAD

**Issue by lclkent** [HobbitLong/SupContrast#19]
Hi thanks for the great work! It seems that the numbers of positives described in the paper is a critical parameters, but the train_loader code isn't specified of this parameter Nyi. Does that means it is ok to randomly sample images of different class?

**Comment by lclkent** [comment]
I am trying this approach on my own dataset of a binary classification task, the data is unbalanced. My loss for the pretraining stage is always around 10.00, I wonder it's because of the unbalanced data and may relate to Nyi mentioned in the paper. Can someone kindly help me out? Appreciated.
